### ISSUE TITLE
OpenBazaar on fedora 26 fails to match architecture on a x86_64 system while i386 runs

### ISSUE THREAD

**Issue by ghost** [OpenBazaar/openbazaar-desktop#985]
Brief Description: 
Unable to install openbazaar, error is "does not have compatible architecture" despite architecture matching. i386 however prompts install.  

Operating System (OS and version): Fedora Workstation 26
OpenBazaar version (shown on `About OpenBazaar` page in menu): 
Hardware: Laptop, Lenovo Yoga 

Reproducible (Always / Almost Always / Sometimes / Rarely / Couldn't Reproduce): 
Always

Steps to reproduce:
1. Download amd64 package for linux from https://www.openbazaar.org/download/
2. Run command sudo dnf install openbazaar2-2.0.13.amd64.rpm

Observed Behavior:
Error: 
 Problem: conflicting requests
  - package openbazaar2-2.0.13-1.amd64 does not have a compatible architecture

x86_64

uname -m: 
x86_64

Expected Behavior:
Installing normally 

Additional info (links, images, etc go here):
Using the i386 package

Dependencies resolved.
===============================================================================================================================================================================================================================
 Package                                                         Arch                                      Version                                                       Repository                                       Size
===============================================================================================================================================================================================================================
Installing:
 openbazaar2                                                     i386                                      2.0.13-1                                                      @commandline                                    111 M
Installing dependencies:
 dbusmenu-qt                                                     x86_64                                    0.9.3-0.12.20150604.fc26                                      fedora                                           87 k
 gdbm-devel                                                      x86_64                                    1.13-1.fc26                                                   fedora                                           62 k
 libdb-devel                                                     x86_64                                    5.3.28-21.fc26                                                fedora                                           43 k
 libmng                                                          x86_64                                    2.0.3-4.fc26                                                  fedora                                          179 k
 libpng12                                                        x86_64                                    1.2.57-1.fc26                                                 fedora                                          157 k
 perl-Algorithm-Diff                                             noarch                                    1.1903-6.fc26                                                 fedora                                           50 k
 perl-Archive-Tar                                                noarch                                    2.26-1.fc26                                                   fedora                                           77 k
 perl-Archive-Zip                                                noarch                                    1.59-3.fc26                                                   fedora                                          105 k
 perl-B-Lint                                                     noarch                                    1.20-8.fc26                                                   fedora                                           23 k
 perl-CGI                                                        noarch                                    4.36-1.fc26                                                   fedora                                          206 k
 perl-CPAN                                                       noarch                                    2.16-1.fc26                                                   fedora                                          552 k
 perl-CPAN-Meta                                                  noarch                                    2.150010-2.fc26                                               fedora                                          189 k
 perl-CPAN-Meta-Requirements                                     noarch                                    2.140-7.fc26                                                  fedora                                           35 k
 perl-CPAN-Meta-YAML                                             noarch                                    0.018-367.fc26                                                fedora                                           32 k
 perl-Class-ISA                                                  noarch                                    0.36-1019.fc26                                                fedora                                           13 k
 perl-Compress-Bzip2                                             x86_64                                    2.26-1.fc26                                                   fedora                                           70 k
 perl-Compress-Raw-Bzip2                                         x86_64                                    2.074-1.fc26                                                  fedora                                           37 k
 perl-Compress-Raw-Zlib                                          x86_64                                    2.074-1.fc26                                                  fedora                                           65 k
 perl-Data-Dump                                                  noarch                                    1.23-4.fc26                                                   fedora                                           35 k
 perl-Data-OptList                                               noarch                                    0.110-3.fc26                                                  fedora                                           30 k
 perl-Data-Section                                               noarch                                    0.200006-8.fc26                                               fedora                                           28 k
 perl-Devel-Size                                                 x86_64                                    0.80-6.fc26                                                   fedora                                           44 k
 perl-Digest                                                     noarch                                    1.17-367.fc26                                                 fedora                                           25 k
 perl-Digest-HMAC                                                noarch                                    1.03-14.fc26                                                  fedora                                           18 k
 perl-Digest-MD5                                                 x86_64                                    2.55-3.fc26                                                   fedora                                           34 k
 perl-Digest-SHA                                                 x86_64                                    1:5.96-2.fc26                                                 fedora                                           62 k
 perl-Encode-Locale                                              noarch                                    1.05-6.fc26                                                   fedora                                           20 k
 perl-Env                                                        noarch                                    1.04-366.fc26                                                 fedora                                           19 k
 perl-ExtUtils-CBuilder                                          noarch                                    1:0.280225-366.fc26                                           fedora                                           46 k
 perl-ExtUtils-Command                                           noarch                                    7.24-3.fc26                                                   updates                                          17 k
 perl-ExtUtils-Install                                           noarch                                    2.04-367.fc26                                                 fedora                                           44 k
 perl-ExtUtils-MM-Utils                                          noarch                                    7.24-3.fc26                                                   updates                                          14 k
 perl-ExtUtils-MakeMaker                                         noarch                                    7.24-3.fc26                                                   updates                                         295 k
 perl-ExtUtils-Manifest                                          noarch                                    1.70-366.fc26                                                 fedora                                           35 k
 perl-ExtUtils-ParseXS                                           noarch                                    1:3.31-368.fc26                                               fedora                                           81 k
 perl-File-CheckTree                                             noarch                                    4.42-298.fc26                                                 fedora                                           21 k
 perl-File-HomeDir                                               noarch                                    1.00-13.fc26                                                  fedora                                           60 k
 perl-File-Listing                                               noarch                                    6.04-14.fc26                                                  fedora                                           16 k
 perl-File-Which                                                 noarch                                    1.21-3.fc26                                                   fedora                                           22 k
 perl-HTML-Parser                                                x86_64                                    3.72-5.fc26                                                   fedora                                          115 k
 perl-HTML-Tagset                                                noarch                                    3.20-28.fc26                                                  fedora                                           22 k
 perl-HTTP-Cookies                                               noarch                                    6.01-14.fc26                                                  fedora                                           28 k
 perl-HTTP-Date                                                  noarch                                    6.02-15.fc26                                                  fedora                                           17 k
 perl-HTTP-Message                                               noarch                                    6.11-4.fc26                                                   fedora                                           87 k
 perl-HTTP-Negotiate                                             noarch                                    6.01-14.fc26                                                  fedora                                           20 k
 perl-IO-Compress                                                noarch                                    2.074-1.fc26                                                  fedora                                          255 k
 perl-IO-HTML                                                    noarch                                    1.001-7.fc26                                                  fedora                                           26 k
 perl-IO-Zlib                                                    noarch                                    1:1.10-390.fc26                                               fedora                                           72 k
 perl-IPC-Cmd                                                    noarch                                    1:0.98-1.fc26                                                 fedora                                           41 k
 perl-IPC-System-Simple                                          noarch                                    1.25-12.fc26                                                  fedora                                           41 k
 perl-JSON-PP                                                    noarch                                    2.94000-1.fc26                                                fedora                                           65 k
 perl-LWP-MediaTypes                                             noarch                                    6.02-11.fc26                                                  fedora                                           27 k
 perl-Locale-Codes                                               noarch                                    3.42-2.fc26                                                   fedora                                          301 k
 perl-Locale-Maketext                                            noarch                                    1.28-2.fc26                                                   fedora                                           97 k
 perl-Locale-Maketext-Simple                                     noarch                                    1:0.21-390.fc26                                               fedora                                           70 k
 perl-MRO-Compat                                                 noarch                                    0.13-1.fc26                                                   fedora                                           22 k
 perl-Math-BigInt                                                noarch                                    1.9998.11-1.fc26                                              fedora                                          193 k
 perl-Math-Complex                                               noarch                                    1.59-390.fc26                                                 fedora                                          100 k
 perl-Module-Build                                               noarch                                    2:0.42.24-1.fc26                                              fedora                                          270 k
 perl-Module-CoreList                                            noarch                                    1:5.20170621-1.fc26                                           fedora                                           80 k
 perl-Module-Load                                                noarch                                    1:0.32-366.fc26                                               fedora                                           17 k
 perl-Module-Load-Conditional                                    noarch                                    0.68-2.fc26                                                   fedora                                           23 k
 perl-Module-Metadata                                            noarch                                    1.000033-2.fc26                                               fedora                                           43 k
 perl-Module-Pluggable                                           noarch                                    2:5.2-4.fc26                                                  fedora                                           32 k
 perl-NTLM                                                       noarch                                    1.09-14.fc26                                                  fedora                                           22 k
 perl-Net-HTTP                                                   noarch                                    6.16-1.fc26                                                   fedora                                           41 k
 perl-Net-Ping                                                   noarch                                    2.43-390.fc26                                                 fedora                                           86 k
 perl-Package-Generator                                          noarch                                    1.106-8.fc26                                                  fedora                                           25 k
 perl-Params-Check                                               noarch                                    1:0.38-366.fc26                                               fedora                                           22 k
 perl-Params-Util                                                x86_64                                    1.07-17.fc26                                                  fedora                                           41 k
 perl-Perl-OSType                                                noarch                                    1.010-4.fc26                                                  fedora                                           27 k
 perl-Pod-Checker                                                noarch                                    4:1.73-2.fc26                                                 fedora                                           31 k
 perl-Pod-Html                                                   noarch                                    1.22.01-390.fc26                                              fedora                                           79 k
 perl-Pod-LaTeX                                                  noarch                                    0.61-299.fc26                                                 fedora                                           39 k
 perl-Pod-Parser                                                 noarch                                    1.63-367.fc26                                                 fedora                                          106 k
 perl-Pod-Plainer                                                noarch                                    1.04-4.fc26                                                   fedora                                           12 k
 perl-Software-License                                           noarch                                    0.103012-4.fc26                                               fedora                                          126 k
 perl-Storable                                                   x86_64                                    1:2.56-368.fc26                                               fedora                                           83 k
 perl-Sub-Exporter                                               noarch                                    0.987-11.fc26                                                 fedora                                           71 k
 perl-Sub-Install                                                noarch                                    0.928-10.fc26                                                 fedora                                           25 k
 perl-Sys-Syslog                                                 x86_64                                    0.35-2.fc26                                                   fedora                                           47 k
 perl-Test-Harness                                               noarch                                    3.39-1.fc26                                                   fedora                                          276 k
 perl-Test-Simple                                                noarch                                    1:1.302086-1.fc26                                             fedora                                          427 k
 perl-Text-Diff                                                  noarch                                    1.44-3.fc26                                                   fedora                                           43 k
 perl-Text-Glob                                                  noarch                                    0.11-1.fc26                                                   fedora                                           16 k
 perl-Text-Soundex                                               x86_64                                    3.05-4.fc26                                                   fedora                                           29 k
 perl-Text-Template                                              noarch                                    1.47-1.fc26                                                   fedora                                           66 k
 perl-Text-Unidecode                                             noarch                                    1.30-2.fc26                                                   fedora                                          147 k
 perl-Time-HiRes                                                 x86_64                                    1.9742-1.fc26                                                 fedora                                           54 k
 perl-TimeDate                                                   noarch                                    1:2.30-10.fc26                                                fedora                                           51 k
 perl-Try-Tiny                                                   noarch                                    0.28-2.fc26                                                   fedora                                           42 k
 perl-WWW-RobotRules                                             noarch                                    6.02-15.fc26                                                  fedora                                           21 k
 perl-XML-LibXML                                                 x86_64                                    1:2.0129-1.fc26                                               fedora                                          367 k
 perl-XML-NamespaceSupport                                       noarch                                    1.11-20.fc26                                                  fedora                                           22 k
 perl-XML-SAX                                                    noarch                                    0.99-18.fc26                                                  fedora                                           63 k
 perl-XML-SAX-Base                                               noarch                                    1.09-1.fc26                                                   fedora                                           33 k
 perl-autodie                                                    noarch                                    2.29-367.fc26                                                 fedora                                           96 k
 perl-devel                                                      x86_64                                    4:5.24.1-390.fc26                                             fedora                                          575 k
 perl-inc-latest                                                 noarch                                    2:0.500-6.fc26                                                fedora                                           23 k
 perl-libwww-perl                                                noarch                                    6.23-1.fc26                                                   fedora                                          207 k
 perl-local-lib                                                  noarch                                    2.000023-1.fc26                                               fedora                                           72 k
 perl-version                                                    x86_64                                    5:0.99.18-1.fc26                                              fedora                                           64 k
 python3-pyparsing                                               noarch                                    2.1.10-3.fc26                                                 fedora                                          140 k
 qt                                                              x86_64                                    1:4.8.7-28.fc26                                               fedora                                          4.8 M
 qt-common                                                       noarch                                    1:4.8.7-28.fc26                                               fedora                                          7.3 k
 qt-x11                                                          x86_64                                    1:4.8.7-28.fc26                                               fedora                                           12 M
 qt3                                                             x86_64                                    3.3.8b-69.fc26                                                fedora                                          3.4 M
 redhat-lsb                                                      x86_64                                    4.1-34.fc26                                                   fedora                                           28 k
 redhat-lsb-cxx                                                  x86_64                                    4.1-34.fc26                                                   fedora                                           20 k
 redhat-lsb-desktop                                              x86_64                                    4.1-34.fc26                                                   fedora                                           24 k
 redhat-lsb-languages                                            x86_64                                    4.1-34.fc26                                                   fedora                                           22 k
 redhat-lsb-printing                                             x86_64                                    4.1-34.fc26                                                   fedora                                           20 k
 redhat-lsb-submod-multimedia                                    x86_64                                    4.1-34.fc26                                                   fedora                                           19 k
 systemtap-sdt-devel                                             x86_64                                    3.1-5.fc26                                                    fedora                                           71 k
Installing weak dependencies:
 perl-Module-Runtime                                             noarch                                    0.015-1.fc26                                                  updates                                          24 k
 perl-TermReadKey                                                x86_64                                    2.37-2.fc26                                                   fedora                                           37 k
 sni-qt                                                          x86_64                                    0.2.6-8.fc26                                                  fedora                                           56 k

Transaction Summary
===============================================================================================================================================================================================================================
Install  118 Packages

Total size: 140 M
Total download size: 30 M
Installed size: 490 M
Is this ok [y/N]: 
Operation aborted.

**Comment by nycnikato** [comment]
Truly sad that three is nio flatpak Support. We are still dealing with RPM hell in the 21st century.

**Comment by hoffmabc** [comment]
I have to revisit it but I believe the main reason is because our CI environment cannot build flatpak binaries since it’s too old. It would require manual building, which is a huge pain in the ass. 

I will take a look again to see if things changed.

**Comment by peterbaumgarten** [comment]
I can duplicate on Fedora 27

**Comment by hegjon** [comment]
amd64 should be changed to x86_64:
```
$ sudo dnf install ./openbazaar2-2.0.21.amd64.rpm 
Last metadata expiration check: 0:01:32 ago on Tue 02 Jan 2018 03:46:25 PM CET.
Error: 
 Problem: conflicting requests
  - package openbazaar2-2.0.21-1.amd64 does not have a compatible architecture

$ rpm -qip ./openbazaar2-2.0.21.amd64.rpm
Name        : openbazaar2
Version     : 2.0.21
Release     : 1
Architecture: amd64
Install Date: (not installed)
Group       : Unspecified
Size        : 445866681
License     : MIT
Signature   : (none)
Source RPM  : openbazaar2-2.0.21-1.src.rpm
Build Date  : Mon 18 Dec 2017 03:43:30 PM CET
Build Host  : travis-job-1c38f38a-7155-40cc-89d0-4c78f51a27ea
Relocations : (not relocatable)
URL         : https://github.com/OpenBazaar/openbazaar-desktop#readme
Summary     : Decentralized Peer to Peer Marketplace for Bitcoin
Description :
Decentralized Peer to Peer Marketplace for Bitcoin

$ rpm -qi nodejs
Name        : nodejs
Epoch       : 1
Version     : 8.9.3
Release     : 2.fc27
Architecture: x86_64
Install Date: Fri 22 Dec 2017 04:12:08 PM CET
Group       : Development/Languages
Size        : 19977139
License     : MIT and ASL 2.0 and ISC and BSD
Signature   : RSA/SHA256, Fri 08 Dec 2017 10:04:42 PM CET, Key ID f55e7430f5282ee4
Source RPM  : nodejs-8.9.3-2.fc27.src.rpm
Build Date  : Fri 08 Dec 2017 08:15:05 PM CET
Build Host  : buildvm-18.phx2.fedoraproject.org
Relocations : (not relocatable)
Packager    : Fedora Project
Vendor      : Fedora Project
URL         : http://nodejs.org/
Summary     : JavaScript runtime
Description :
Node.js is a platform built on Chrome's JavaScript runtime
for easily building fast, scalable network applications.
Node.js uses an event-driven, non-blocking I/O model that
makes it lightweight and efficient, perfect for data-intensive
real-time applications that run across distributed devices.
```

**Comment by hoffmabc** [comment]
ok thanks will update the build.

**Comment by jjeffryes** [comment]
This was fixed in #1274
